crux/crux.cabal

```diff
@@ -34,6 +34,8 @@ library
     containers,
     contravariant >= 1.5 && < 1.6,
     crucible,
+    crucible-debug,
+    crucible-syntax,
```

Comment:
Worth mentioning in the changelog that `crux` now incurs some extra dependencies (`crucible-debug` and `crucible-syntax`).